ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/SpacingAroundCurlyRuleTest.kt

```diff
@@ -209,4 +209,71 @@ class SpacingAroundCurlyRuleTest {
                 { 42 }
         """.trimIndent())
     }
+
+    @Test
+    fun `eol comment placed after curly brace`() {
+        assertThat(
+            SpacingAroundCurlyRule().format(
+                """
+                class MyClass()// a comment
+                {
+                    val x = 0
+                }
+                """.trimIndent()
+            )
+        ).isEqualTo(
+            """
+                class MyClass() {
```

Comment:
I'm wondering if it actually should be 
```
class MyClass() { // a comment
    ...
}
```

Reply:
Yes I think leaving the comment on the same line as the curly brace is preferable to moving it to next line. I just pushed the code.